tests/test_states.py

```diff
@@ -220,6 +247,394 @@ def test_squeezed_coherent(self):
         self.assertAllAlmostEqual(res.flatten(), res_exact.flatten(), delta=self.tol)
 
 
+class PolyQuadExpectationSingleMode(BaseTest):
```

Comment:
Just noticed this now, but the test classes in `test_states.py` all have names without `Test` at the end, while in all other test files (that I'm aware of), we use the convention `SomeKindOfTest(ParentTest)`

Reply:
Let's leave this for a future PR, and perhaps make an issue for it?

Reply:
This was quick, just took care of it